docs/sphinx/examples/python/quantinuum

```diff
@@ -0,0 +1,49 @@
+import cudaq
+
+# You only have to set the target once! No need to redefine it
+# for every execution call on your kernel.
+# To use different targets in the same file, you must update
+# it via another call to `cudaq.set_target()`
+cudaq.set_target("quantinuum")
+
+# Create the kernel we'd like to execute on Quantinuum
+kernel = cudaq.make_kernel()
+qubits = kernel.qalloc(2)
+kernel.h(qubits[0])
+kernel.cx(qubits[0], qubits[1])
+kernel.mz(qubits[0])
+kernel.mz(qubits[1])
+# Print out the kernel instructions before submitting.
+print(kernel)
```

Comment:
I wouldn't include that in the example. What is the purpose of printing the MLIR here? This isn't even what gets submitted; this is pre-compilation IR. I am not sure what the user is supposed to get from it.